docs/docs-site/content/developer/development/_index.md

```diff
@@ -508,7 +508,7 @@ Run the user interface tests:
 
 In the pyproject.toml file, we define the configuration for pytest, including settings to enable pytest to discover and execute the unit tests.
 
-For pytest to recognize test files, you should follow the naming convention `*_test.py` or `test_*.py`. Following the pytest framework [convention](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery) for test discovery is recommended. Pytest will gather test items according to the following rules:
```

Comment:
any specific reason to remove this link?

Reply:
Its giving 403 error